python/triton/compiler.py

```diff
@@ -1063,12 +1063,13 @@ def ptx_get_version(cuda_version) -> int:
 def path_to_ptxas():
     base_dir = os.path.dirname(__file__)
     paths = [
+        os.path.join(os.environ.get("TRITON_PTXAS_PATH", ""), "ptxas"),
```

Comment:
A problen with that is that it will grab `ptxas` in the current dir over the packaged one when `TRITON_PTXAS_PATH` is not defined